Gulpfile.js

```diff
@@ -158,74 +157,73 @@ gulp.task('css:app', gulp.series('css:application', 'css:lint'));
+      './node_modules/@bower_components/angular-strap/dist/modules/modal.tpl.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/alert.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/alert.tpl.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/popover.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/popover.tpl.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/collapse.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/parse-options.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/typeahead.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/typeahead.tpl.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/select.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/select.tpl.js',
+
+      './node_modules/@bower_components/angular-strap/dist/modules/date-parser.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/date-formatter.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/datepicker.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/datepicker.tpl.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/timepicker.js',
+      './node_modules/@bower_components/angular-strap/dist/modules/timepicker.tpl.js',
+
+      './node_modules/@bower_components/angular-breadcrumb/release/angular-breadcrumb.js',
+
+      './node_modules/@bower_components/ngstorage/ngStorage.js',
+      './node_modules/@bower_components/angular-loading-bar/build/loading-bar.js',
+
+      './node_modules/@bower_components/d3/d3.js',
+      './node_modules/@bower_components/d3-timeline/src/d3-timeline.js',
+      './node_modules/@bower_components/lodash/lodash.js',
+      './node_modules/@bower_components/graphlib/dist/graphlib.core.js',
+      './node_modules/@bower_components/dagre/dist/dagre.core.js',
+      './node_modules/@bower_components/dagre-d3/dist/dagre-d3.core.js',
+      './node_modules/@bower_components/moment/moment.js',
+      './node_modules/@bower_components/angular-moment/angular-moment.js',
+      './node_modules/@bower_components/angular-bootstrap/ui-bootstrap-tpls.js',
+
+      './node_modules/@bower_components/node-uuid/uuid.js',
+
+      './node_modules/@bower_components/angular-cookies/angular-cookies.min.js',
       './node_modules/redux/dist/redux.min.js',
       './node_modules/redux-thunk/dist/redux-thunk.min.js',
-      './bower_components/ace-builds/src-min-noconflict/ace.js',
-      './bower_components/angular-ui-ace/ui-ace.js',
-      './bower_components/jsPlumb/dist/js/jsPlumb-2.0.6-min.js',
-      './bower_components/angular-gridster/dist/angular-gridster.min.js',
-      './bower_components/angular-cron-jobs/dist/angular-cron-jobs.min.js',
-      './bower_components/angularjs-dropdown-multiselect/dist/angularjs-dropdown-multiselect.min.js',
```

Comment:
is this also removed?

Reply:
yes, this component is not in use anymore. It has been replaced by a newer `multiselect` component written in react.